scripts/generate-textlint-runtime.py

```
@@ -0,0 +1,45 @@
+
+import json
+import pathlib
+import sys
+from typing import Sequence
+
+
+def main(argv: Sequence[str]) -> None:
+    if len(argv) < 3:
+        raise ValueError(
+            f"expected 3 arguments (src, prh, dest), got {len(argv)}"
+        )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)